.github/workflows/scripts/e2e.maven.default.verify.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# shellcheck source=/dev/null
+source "./.github/workflows/scripts/e2e-verify.common.sh"
+
+# Input variables
+EXPECTED_ARTIFACT_OUTPUT=${EXPECTED_ARTIFACT_OUTPUT:-}
+PROVENANCE_DIR=${PROVENANCE_DIR:-}
+GITHUB_REF_NAME=${GITHUB_REF_NAME:-}
+GITHUB_REF=${GITHUB_REF:-}
+RUNNER_DEBUG=${RUNNER_DEBUG:-}
```

Comment:
nit: add POMXML to script input
```suggestion
POMXML=${POMXML:-} # specified in the e2e test yaml
RUNNER_DEBUG=${RUNNER_DEBUG:-}
```